server/routes/index.ts

```diff
@@ -148,6 +148,7 @@ router.use('/media', isAuthenticated(), mediaRoutes);
 router.use('/person', isAuthenticated(), personRoutes);
 router.use('/collection', isAuthenticated(), collectionRoutes);
 router.use('/service', isAuthenticated(), serviceRoutes);
+router.use(trackUserActivity);
```

Comment:
This should probably be higher up alongside our other middleware. Also, I am not sure how I feel about us running a user update on every single API request. I think just doing this in our usual auth check would be enough.